tests/test-backend-ops.cpp

```diff
@@ -720,7 +721,7 @@ struct test_dup : public test_case {
     ggml_tensor * build_graph(ggml_context * ctx) override {
         ggml_tensor * src = ggml_new_tensor(ctx, type, 4, ne.data());
         if (_use_permute) {
-            src = ggml_permute(ctx, src, permute[0], permute[1], permute[2], permute[3]);
+            src = ggml_permute(ctx, src, int(permute[0]), int(permute[1]), int(permute[2]), int(permute[3]));
```

Comment:
Change the declaration of `permute` to int instead.

Reply:
The good way would be to adapt the declarations of such vectors and/or functions in general, I'd suggest to make it as a next step.